components/dashboards-web-component/src/designer/DashboardDesigner.css

```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This is a newly added file, so year should be 2018

Reply:
No this was there before

Reply:
Ah, you just added the license header. Then OK.